contracts/libs/data-structures/IncrementalMerkleTree.sol

```diff
@@ -149,6 +213,14 @@ library IncrementalMerkleTree {
         IMT _tree;
     }
 
+    /**
+     * @notice The Address Incremental Merkle Tree constructor, creates a tree instance with the
+     * given size, O(1) complex
```

Comment:
Let's not specify the complexity here. The creation of an array in Solidity in O(n)...